### PR TITLE
addpkg: lftp

### PR DESCRIPTION
**Pull request by Nightwulf**
Original author contacted to update config.guess/config.sub. Added those two files and modified PKGBUILD zo perform autoupdate/autoreconf until that is done.

### CODE REVIEW

**Comment by r-value** [comment]
+ please submit a patch instead of a full `PKGBUILD` file
+ usually a line of `autoreconf -fiv` can solve the problem instead of manually overwrite them
+ you can leave a link to your contact threads if possible

**Comment by r-value** [comment]
#3199 could be a good example for you

**Comment by Nightwulf** [comment]
original author contacted about the issue: https://github.com/lavv17/lftp/issues/717

File: lftp/riscv64.patch [hunk 1]
```diff
@@ -0,0 +1,21 @@
+diff --git a/PKGBUILD b/PKGBUILD
+index df5e9c3..9ed745c 100644
+--- a/PKGBUILD
++++ b/PKGBUILD
```
Comment (Xeonacid) [inline lftp/riscv64.patch:4]: Please remove the a/ b/ as other patches do.

**Review by r-value — APPROVED** [review]
LGTM

**Review by r-value — COMMENTED** [review]
It could be better if you can tidy your commit history and squash the commits into a single one :)

**Comment by Nightwulf** [comment]
done

**Comment by felixonmars** [comment]
Thanks!

**Pull request merged**